docs/releases/v1.5.0.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+
+## Changelog
+
+Changes between `1.4.1` and this release: `1.5.0`
+
+- Refactor `dual-nginx` package to use the base of the `nginx` *(simple)* deployment.
+
+### Upgrade path
+
+If you already have deployed the `dual-nginx` package you should follow this procedure:
+
+
+Within this repository clonned locally run:
```

Comment:
Typo: clonned.

Reply:
Shouldn't the upgrade instructions be given from an "infra" repository POV?

Reply:
Uhmmmmmmmm. Maybe.........

Reply:
Done!